lib/tumblr.js

```diff
@@ -478,6 +478,23 @@ function getRequest(requestGet, credentials, baseUrl, apiPath, requestOptions, p
         params.api_key = credentials.consumer_key;
     }
 
+    // if given /v2/user/info, remove the /v2
+    if (apiPath.substr(0, 3) === '/v2') {
```

Comment:
Not a big deal, but could probably use [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) here.

Reply:
ah yeah, i am so far behind what things i should expect to be available to me in javascript 😅

Reply:
It's generally safe to assume nothing is free in JavaScript 😆

Reply:
in this case i think i'll leave it as-is as to not drop IE support, i don't wanna cause that headache here

Reply:
Do you wanna refactor this into a method since you're already doing it twice here? Something like `getApiPath = (path) => path.substr(0, 3) === '/v2' ? path.substr(3) : path`

Reply:
I can't imagine this library ever gets used on IE, since it requires node libraries and utils? So the relevant version would be 0.12 behind the harmony flag or 4.0.0 without the flag

Reply:
(so imo we're well clear to go ahead and use startsWith)

Reply:
You're totally right. None of this stuff works in a browser anyway.

Reply:
oh, lol, yeah, good points